apps/src/netsim/NetSimRouterTab.js

```diff
@@ -21,12 +21,14 @@ var NetSimRouterStatsTable = require('./NetSimRouterStatsTable');
  * Generator and controller for router information view.
  * @param {jQuery} rootDiv - Parent element for this component.
  * @param {netsimLevelConfiguration} levelConfig
- * @param {function} bandwidthChangeCallback
- * @param {function} memoryChangeCallback
+ * @param {Object} callbacks
+ * @param {function} callbacks.bandwidthSliderChangeCallback
```

Comment:
I'm reading this as, these are required to all be present, which makes sense, even though it looks like a caller could  operate it without them